src/browser/HostInstaller.cpp

```diff
@@ -40,7 +40,7 @@ HostInstaller::HostInstaller()
     , TARGET_DIR_VIVALDI("/Library/Application Support/Vivaldi/NativeMessagingHosts")
     , TARGET_DIR_TOR_BROWSER("/Library/Application Support/TorBrowser-Data/Browser/Mozilla/NativeMessagingHosts")
     , TARGET_DIR_BRAVE("/Library/Application Support/BraveSoftware/Brave-Browser/NativeMessagingHosts")
-#elif defined(Q_OS_LINUX)
+#elif defined(Q_OS_UNIX)
```

Comment:
Shouldn't think be changed to `#elif defined(Q_OS_LINUX) || defined(Q_OS_UNIX)`
Perhaps even moving this to the end as an `#else` catch-all.

Reply:
Hmm, I was assuming that Q_OS_UNIX is defined on Linux too since it is already used in several other places. Is that not the case?

Reply:
Moved it to an `#else`instead.

Reply:
The Qt docs state:
> Q_OS_UNIX: Defined on Any UNIX BSD/SYSV system.

Reply:
Although after looking at the Qt headers it appears Q_OS_UNIX is defined whenever Q_OS_WIN is not defined.

Reply:
The #else clause has been added, so let me mark this as resolved.